### PR TITLE
Remove leftovers after the isys module removal

### PR DESCRIPTION
**Pull request by VladimirSlavik**
There's always more to remove.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -356,6 +356,6 @@ def __getattr__(cls, name):
     def __getitem__(cls, key):
         return cls.__getattr__(key)
 
-MOCK_MODULES = ['_isys']
+MOCK_MODULES = []
 for mod_name in MOCK_MODULES:
     sys.modules[mod_name] = Mock()
```
Comment (VladimirSlavik) [inline docs/conf.py:361]: I wonder if this is needed at all.

Comment (poncovka) [reply docs/conf.py:361]: I would drop that.

**Review by poncovka — APPROVED** [review]
Thanks!

File: docs/conf.py (hunk 1 above)
Comment (VladimirSlavik) [reply docs/conf.py]: Thanks, agreed.

**Pull request merged**